app/apps/addresses/models.py

```diff
@@ -64,34 +69,43 @@ def __str__(self):
             )
         return self.bag_id
 
-    def get(bag_id):
+    def get_or_create(bag_id):
```

Comment:
Het is verwarrend als een method dezelfde naam heeft als een bekende method.
`get_or_create_by_bag_id` is wellicht duidelijker.